spyder/plugins/projects/widgets/tests/test_qcookiecutter.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
```

Comment:
If you decide to change the name of `qcookiecutter.py`, please don't forget to rename this one too.

Reply:
No change